### DISCUSSION TITLE
Running Nginx getting directories are missing.

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#1669] [Help Needed]
Hello, so I'm able to install nginx and even setup a host (able to pull letsencrypt certificate). But when I click on the host or even type in the host I get the following errors

2023/08/06 23:25:34 [error] 6777#6777: *136 open() "/var/www/html/.env" failed (2: No such file or directory), client: 147.>
2023/08/06 23:55:38 [alert] 6777#6777: 512 worker_connections are not enough
2023/08/06 23:55:38 [error] 6777#6777: *733 recv() failed (104: Connection reset by peer) while reading response header fro>
2023/08/06 23:56:32 [error] 6777#6777: *735 "/var/www/html/geoserver/web/index.html" is not found (2: No such file or direc>
2023/08/07 00:49:55 [alert] 6866#6866: 512 worker_connections are not enough
2023/08/07 00:49:55 [error] 6866#6866: *1259 recv() failed (104: Connection reset by peer) while reading response header fr>

Not sure what I'm missing with this. Thanks in advance for your help.

**Comment by tteck** [comment]
Are you referencing Nginx Proxy Manager?

**Comment by Dvalin21** [comment]
Yes, sorry. It was early in the morning when I made this post
